### PR TITLE
0.45 release notes

### PR DESCRIPTION
**Pull request by vladvino**

### CODE REVIEW

File: changelogs/api-management-service.md [hunk 1]
```diff
@@ -1,5 +1,38 @@
...
+## Release - API Management service: June, 2024
+
+###  ‼️ Breaking changes ‼️
+
+- URL properties in the [Backend](https://learn.microsoft.com/rest/api/apimanagement/backend/create-or-update) entity no longer accept URLs containing query parameters. We've made the change to fix a bug. Please use [`set-query-parameter`](https://learn.microsoft.com/azure/api-management/set-query-parameter-policy) policy and [`backend.credentials.query`](https://learn.microsoft.com/rest/api/apimanagement/backend/create-or-update#backendcredentialscontract) to set query parameters.
+
+### New features, improvements, and changes
+
+- HTTP/2 connections are now drained and gracefully terminated before node restarts
+- We added support for decryption keys to the [`validate-azure-ad-token`](https://learn.microsoft.com/azure/api-management/validate-jwt-policy) policy
```
Comment (mikebudzynski) [inline changelogs/api-management-service.md:12]: The correct link is https://learn.microsoft.com/azure/api-management/validate-azure-ad-token-policy

File: changelogs/api-management-service.md [hunk 2]
```diff
@@ -1,5 +1,38 @@
...
+- [`azure-openai-semantic-cache-lookup`](https://learn.microsoft.com/azure/api-management/azure-openai-semantic-cache-lookup-policy) and [`azure-openai-semantic-cache-store`](https://learn.microsoft.com/azure/api-management/azure-openai-semantic-cache-store-policy) policies now work in both V2 and Classic tiers.
```
Comment (mikebudzynski) [inline changelogs/api-management-service.md:13]: Unnecessary full stop at the end

File: changelogs/api-management-service.md [hunk 3]
```diff
@@ -1,5 +1,38 @@
...
+- We've imprioved promt token estimation accuracy for GPT-4 models in the [`azure-openai-token-limit `](https://learn.microsoft.com/azure/api-management/azure-openai-token-limit-policy) policy
+
+### Fixes
+
+- We fixed an issue which caused GraphQL resolver runtime errors when primitive type fileds had null value
+- We fixed a bug and stopped the fields of type array-of-objects from causing GraphQL resolver runtime errors
+- Load balancer feature ...
+- Expressions are now work as expected for in the `<audience/>` elements in the [`validate-jwt`](https://learn.microsoft.com/azure/api-management/validate-jwt-policy) policy
+- We fixed a bug that caused XML entities to be erroneously unescaped in the policy documents
```
Comment (mikebudzynski) [inline changelogs/api-management-service.md:22]: 1. Not XML entities, but text values of XML elements.
2. Decoded, not unescaped.

File: changelogs/api-management-service.md [hunk 4]
```diff
@@ -1,5 +1,35 @@
 # Azure API Management service changelog
 
+## Release - API Management service: June, 2024
+
+### New features, improvements, and changes
+
+- HTTP/2 connections are now drained and gracefully terminated before node restarts
+- We added support for decryption keys to the [`validate-azure-ad-token`](https://learn.microsoft.com/azure/api-management/validate-azure-ad-token-policy) policy
+- [`azure-openai-semantic-cache-lookup`](https://learn.microsoft.com/azure/api-management/azure-openai-semantic-cache-lookup-policy) and [`azure-openai-semantic-cache-store`](https://learn.microsoft.com/azure/api-management/azure-openai-semantic-cache-store-policy) policies now work in both V2 and Classic tiers.
+- We've imprioved promt token estimation accuracy for GPT-4 models in the [`azure-openai-token-limit `](https://learn.microsoft.com/azure/api-management/azure-openai-token-limit-policy) policy
```
Comment (simonkurtz-MSFT) [inline changelogs/api-management-service.md:10]: Typos on _improved_ and _prompt_

File: changelogs/api-management-service.md [hunk 5]
```diff
@@ -1,5 +1,35 @@
...
+
+### Fixes
+
+- We fixed an issue which caused GraphQL resolver runtime errors when primitive type fileds had null value
```
Comment (simonkurtz-MSFT) [inline changelogs/api-management-service.md:14]: Typo on _fields_

File: changelogs/api-management-service.md [hunk 6]
```diff
@@ -1,5 +1,35 @@
...
+- We fixed a bug and stopped the fields of type array-of-objects from causing GraphQL resolver runtime errors
+- Expressions are now work as expected for in the `<audience/>` elements in the [`validate-jwt`](https://learn.microsoft.com/azure/api-management/validate-jwt-policy) policy
```
Comment (simonkurtz-MSFT) [inline changelogs/api-management-service.md:16]: Change to _Expressions now work as expected in the `<audience/>` elements ..._

**Review by simonkurtz-MSFT — CHANGES_REQUESTED** [review]

File: changelogs/api-management-service.md [hunk 7]
```diff
@@ -1,5 +1,37 @@
...
+
+- HTTP/2 connections are now drained and gracefully terminated before node restarts
+- We added support for decryption keys to the [`validate-azure-ad-token`](https://learn.microsoft.com/azure/api-management/validate-azure-ad-token-policy) policy
+- [`azure-openai-semantic-cache-lookup`](https://learn.microsoft.com/azure/api-management/azure-openai-semantic-cache-lookup-policy) and [`azure-openai-semantic-cache-store`](https://learn.microsoft.com/azure/api-management/azure-openai-semantic-cache-store-policy) policies now work in both V2 and Classic tiers.
+- We've improved prompt token estimation accuracy for GPT-4 models in the [`azure-openai-token-limit `](https://learn.microsoft.com/azure/api-management/azure-openai-token-limit-policy) policy
+- We've made [request tracing](https://learn.microsoft.com/azure/api-management/api-management-howto-api-inspector) more secure and it's now available in both Classic and V2 tiers
+- Now customers can use Azure portal to [migrate](https://learn.microsoft.com/azure/api-management/migrate-stv1-to-stv2-vnet?#trigger-migration-of-a-network-injected-api-management-instance) their stv1 service instances to stv2
```
Comment (ferantoMSFT) [inline changelogs/api-management-service.md:12]: @vladvino please let's change "their stv1 service instances to stv2" to "their VNet injected stv1 service instances to stv2"... emphasis on VNet Injected.

**Pull request merged**